src/main/webapp/script.js

```diff
@@ -0,0 +1,33 @@
+function listElections() {
+  let stateSelected = document.getElementById('select-state').value;
+  let nationalElections = [];
+  let stateElections = [];
+
+  if (stateSelected === null) {
+    return;
+  }
+
+  fetch('/election')
+    .then(response => response.json())
+    .then((textResponse) => {
+      let electionList = JSON.parse(textResponse).elections;
+      electionList.forEach((election) => {
+        if (election.ocdDivisionId == stateSelected) {
+          // TODO: update ocdDivisionId to cater to state vs. national elections
+          stateElections.append(election);
+        }
+      });
+      
+      buildElectionListElements(stateElections);
+  });
+}
+
+function buildElectionListElements(elections) {
+  const electionListContainerElement = document.getElementById('election-list-content');
+
+  const electionsInStateHeadingElement = document.createElement('h3');
+  electionsInStateHeadingElement.innerHTML = 'Elections coming up in Washington';
```

Comment:
I assume "Washington" is going to get replaced later? Can you either have a comment indicating where it'll get a replaced, or a TODO to dynamically populate it?

Reply:
Sorry, old commit was using hard-coded information to check that the elements were showing up as expected. The Washington is gone now!